auth/cert/sslutils_windows.go

```diff
@@ -1,3 +1,4 @@
+//go:build windows
```

Comment:
Is this relevant to the PR?

Reply:
It is not, but this is something go 1.17 automatically modifies. It did the same for v2 as well, after we started releasing it with go 1.17.
I'll make sure that v1 is also moved to go 1.17.